static/styles/zulip.css

```diff
@@ -829,6 +829,15 @@ td.pointer {
         }
     }
 
+    /* We override all `div` properties
+     * here via `> div` above, but we don't want
+     * them to apply for tooltips.
+     */
+    [data-tippy-root] {
+        word-wrap: unset;
+        width: max-content;
+    }
```

Comment:
A better fix is probably to replace the `> div` block with an `> dev.message_control` block in a prep commit, and then add that CSS class on each message control -- as we've seen here, it's kinda a bad idea to have logic apply to every div element inside another.

Reply:
Done, it still looks like it needs those properties since width is being restricted for the tooltip without it.

Reply:
Yeah, OK, that makes sense to me.  Can you fix the comment to not line-wrap oddly?  It can fit one one line.  I'd also spell it "Tooltips should not follow the width restrictions of their parent element."